### DISCUSSION TITLE
How can I use coil-compose with an svg?

### DISCUSSION THREAD

**Discussion by ColtonIdle** [coil-kt/coil#876] [Q&A]
Currently, with the code taken from the docs, nothing will render when using coil-compose.

**Comment by colinrtwhite** [comment]
It should work automatically if you add the `SvgDecoder` to your `ImageLoader`. I'd try [enabling logging](https://coil-kt.github.io/coil/faq/#how-do-i-enable-logging) to see if there are exceptions.

**Reply by ColtonIdle** [reply]
It did work. Just had to add the coil-svg dependency since I was exclusively using `io.coil-kt:coil-compose-base`

Hopefully that sounds right to you. Cheers!
